src/main/java/org/jabref/model/entry/types/BibtexEntryTypeDefinitions.java

```diff
@@ -177,8 +177,22 @@
             .withImportantFields(StandardField.MONTH, StandardField.YEAR)
             .build();
 
+
+    private static final BibEntryType SOFTWARE = new BibEntryTypeBuilder()
```

Comment:
I think `software` and `dataset` are specific to biblatex and thus shouldn't be added here for the bibtex entry types.